dist/index.js

```diff
@@ -6231,6 +6232,57 @@ function getDefaultGlobOptions() {
         omitBrokenSymbolicLinks: true
     };
 }
+/**
+ * If multiple paths are specific, the least common ancestor (LCA) of the search paths is used as
+ * the delimiter to control the directory structure for the artifact. This function returns the LCA
+ * when given an array of search paths
+ *
+ * Example 1: The patterns `/foo/` and `/bar/` returns `/`
+ *
+ * Example 2: The patterns `~/foo/bar/*` and `~/foo/voo/two/*` and `~/foo/mo/` returns `~/foo`
+ */
+function getMultiPathLCA(searchPaths) {
+    if (searchPaths.length < 2) {
+        throw new Error('At least two search paths must be provided');
+    }
+    const commonPaths = new Array();
+    const splitPaths = new Array();
+    let smallestPathLength = Number.MAX_SAFE_INTEGER;
+    // split each of the search paths using the platform specific separator
+    for (const searchPath of searchPaths) {
+        core_1.debug(`Using search path ${searchPath}`);
+        const splitSearchPath = searchPath.split(path.sep);
```

Comment:
I think you should normalize the `searchPath` first.  What if the search path has `//` in the folder path?

Reply:
I added an extra `normalize` call before spiting the paths just to be safe. From my testing, `getSearchPaths` returns paths that are already normalized so this shouldn't be necessary. Nevertheless, doesn't hurt to include it.